### PR TITLE
App name bug

### PR DESCRIPTION
**Pull request by Tatev321**
app name not showing properly 

after creating the app for the game asset added update tab name

### CODE REVIEW

File: editor/src/com/talosvfx/talos/editor/project2/apps/SceneHierarchyApp.java [hunk 1]
```diff
@@ -53,11 +53,7 @@ public void updateForGameAsset (GameAsset<Scene> gameAsset) {
 
 	@Override
 	public String getAppName () {
-		if (gameAsset != null) {
-			return "Hierarchy - " + gameAsset.nameIdentifier;
-		} else {
-			return "Hierarchy - ";
-		}
+		return "Hierarchy";
```
Comment (Tom-Ski) [inline editor/src/com/talosvfx/talos/editor/project2/apps/SceneHierarchyApp.java:56]: Hierarchy should probably show the scene its editing. You can potentially have many of these

File: editor/src/com/talosvfx/talos/editor/project2/apps/PropertiesPanelApp.java [hunk 2]
```diff
@@ -46,11 +46,7 @@ public void updateForGameAsset (GameAsset<IPropertyHolder> gameAsset) {
 
 	@Override
 	public String getAppName () {
-		if (gameAsset != null) {
-			return "Properties - " + gameAsset.nameIdentifier;
-		} else {
-			return "Properties - ";
-		}
+		return "Properties";
```
Comment (Tom-Ski) [inline editor/src/com/talosvfx/talos/editor/project2/apps/PropertiesPanelApp.java:49]: Properties also is generic, should be specific to the asset its showing

File: editor/src/com/talosvfx/talos/editor/addons/scene/apps/spriteeditor/SpriteEditorApp.java [hunk 3]
```diff
@@ -58,11 +58,7 @@ public void updateForGameAsset (GameAsset<Texture> gameAsset) {
 
     @Override
     public String getAppName () {
-        if (gameAsset != null) {
-            return "Sprite - " + gameAsset.nameIdentifier;
-        } else {
-            return "Sprite - ";
-        }
+        return "Sprite";
```
Comment (Tom-Ski) [inline editor/src/com/talosvfx/talos/editor/addons/scene/apps/spriteeditor/SpriteEditorApp.java:61]: We should show the name of the sprite we are editing too

**Review by Tom-Ski — CHANGES_REQUESTED** [review]

**Comment by azakhary** [comment]
is this good to merge @Tom-Ski ?

**Review by Tom-Ski — APPROVED** [review]

**Pull request merged**